markdown/chapters/meandeck/deckconstruction.md

```diff
@@ -0,0 +1,603 @@
+{U}{B} colour pairing with occasionally secondary and tertiary splash colours being
+integrated in. The deck runs a fetchland manabase normally consisting of four copies
+of {{Polluted Delta}} and four to five additional *off-colour* fetches. The *off-colour*
+fetches tend to be the {U} based splash fetch (where a splash is applicable) as the
+majority with ocassional incorporation of the {B} based splash fetch. 
+
+For example: In a Grixis {U}{B}{R} based Doomsday list you might elect to run four
+Polluted Delta, three {{Scalding Tarn}} and one {{Bloodstained Mire}} as your fetchlands 
+of choice.
+
+<row variant="pile">{{!Polluted Delta}} {{!Scalding Tarn}} {{!Bloodstained Mire}}</row>
+
+An eight fetchland configuration is not required with some people opting for running
+nine with a reduced mana-producing land count. Additionally for more controlling
+builds, some may elect to replace some of these with copies of {{Prismatic Vista}}
+in conjunction with an increased basic lands count however these are considered not
+as effective as other fetchlands.
+
+In terms of fetchable lands, {{Underground Sea}} is the dominant dual land used. For
+most straight {U}{B} lists this is a four-of that is run however for less aggressive
+lists or lists with splash colours in this can often be a three of. Some very aggressive
+lists which have started to forgo basic lands entirely are option to run four Seas and
+a copy of {{Watery Grave}} as a pseudo fifth copy. In this instance, if no basic lands
+are being run, the fetchland configuration can be changed to run any combination of
+{U}/x or {B}/x fetchlands. (Diversification is important to avoid {{Pithing Needle}}
+type effects. Conversely, in a more control-based list this can drop down as far as
+only one or two copies of Underground Sea with an Island dominant manabase being 
+favoured with copies of {{Urborg, Tomb of Yawgmoth}} being run as a mana fixer of choice.
+
+If splash colours are being run then it is generally accepted to run one {U}/x dual land
+of the splash colour and one {B}/x dual land of the splash colour.
+
+For example: In a BUG {B}{U}{G} based Doomsday list you might elect to run three
+Underground Seas, one {{Tropical Island}} and one {{Bayou}} as your dual lands of choice.
+
+<row variant="pile">{{!Underground Sea}} {{!Tropical Island}} {{!Bayou}}</row>
+
+The final land type to discuss (for this section) is your basic land count.
+Traditionally only two to three basics were run normally in a 2:1 ratio of
+Island to Swamp. More recently though some are opting to run only one of each,
+some only the Island and others (as mentioned above) are opting to run no
+basic lands at all. If you want to opt for a more control style build then
+you will probably need to look to increase your basic land count to four or 
+five. Normally you wouldn't consider running a splash basic such as a 
+mountain or forest however in the heavier control builds you might consider
```

Comment:
is there another word for "consider", so we dont have to use it twice?

Reply:
Changed.